deploy/cloud-foundry/Jenkinsfile

```diff
@@ -57,6 +60,16 @@ pipeline {
         }
       }
     }
+    stage('Register with ELK') {
```

Comment:
As discussed on Slack, and verified on our Jenkins cluster, the ELK registration command will fail when run from inside the Docker agent.

Set a stage-specific agent so that this command is run on the Jenkins node's native shell.